doc/reference/preseed_yaml_fields.md

```diff
@@ -0,0 +1,78 @@
+(preseed-yaml-file-fields)=
+# Preseed YAML file fields
+
+You can configure a new LXD installation and reconfigure an existing installation with a preseed YAML file.
+
+The preseed YAML file fields are as follows:
+
+```yaml
+config:
+  core.https_address: ""
+  core.trust_password: ""
+  images.auto_update_interval: 6
+
+networks:
+  - config:
+      ipv4.address: auto
+      ipv4.nat: "true"
+      ipv6.address: auto
+      ipv6.nat: "true"
+    description: ""
+    name: lxdbr0
+    type: bridge
+    project: default
+
+storage_pools:
+  - config: {}
+    description: ""
+    name: default
+    driver: zfs
+
```

Comment:
Can you add `storage_volumes`?
This was added in https://github.com/canonical/lxd/pull/12426